src/app/api/common/votes/getVotes.ts

```diff
@@ -18,6 +18,55 @@ const getVotesForDelegate = ({
     page,
   });
 
+async function getSnapshotVotesForDelegate({
+  address,
+  page = 1,
+}: {
+  address: string;
+  page: number;
+}) {
+  const { slug } = Tenant.current();
+  const pageSize = 10;
+
+  const { meta, data: votes } = await paginateResult(
```

Comment:
If we have prisma is there a reason for doing queryRaw here vs just using prisma like an ORM as intended with something like

```
prisma.snapshotVotes.findMany({
  where: {
    dao_slug: slug
  },
  select: {
     ....you get the point
  }
})
```

any perf hit for this or whats the reason for queryRaw? 
personal pref but I find the prisma version easier to read (I'm used to it)
and I don't see why we are using prisma at all if we are just going to subvert it all of the time with queryRaw, right?

Reply:
No, prisma is only there if you want it for some to-be-written access function.

My personal pref is the raw.  And also, my prediction is that we're going to outgrow it completely. Just a question of when.